src/ChannelManager.cpp

```diff
@@ -0,0 +1,210 @@
+// ChannelManager.cpp
+
+// Implements the cChannelManager class which stores and calls channel callbacks
+
+#include "Globals.h"  // NOTE: MSVC stupidness requires this to be the same across all modules
+
+#include <algorithm>
+
+#include "ChannelManager.h"
+#include "ChannelCallback.h"
+#include "ClientHandle.h"
+#include "Entities/Player.h"
+#include "OSSupport/CriticalSection.h"
+#include "World.h"
+
+
+class cChannelBroadcastCallback : public cPlayerListCallback
```

Comment:
How about moving this entire class into the `cChannelManager` class? `cChannelManager::cBroadcastCallback`.